doc/23_Installation_and_Upgrade/03_System_Setup_and_Hosting/08_Multi_Application_Setup.md

```diff
@@ -0,0 +1,23 @@
+# Multi-application setup
+
+## Sessions
+
+When running multiple applications on the same domain, there can be session cookie collisions which prevent you to log in to both systems at the same time.
+
+Imagine you run a Magento app on http://example.org and Pimcore on http://pim.example.org. Then you will have 2 cookies with name `PHPSESSID` (if `session.name` in php.ini is the same for both):
+| Name| Value  | Domain  |  Path | 
+|---|---|---|---|
+| PHPSESSID | 5a9b08750387d9e11c738a2947d93e38   |  .example.org | /  | 
+| PHPSESSID | irqnjh5p96gp2i8iu743ulm32p   |  pim.example.org | /  | 
+
+First one is from Magento, second from Pimcore.
```

Comment:
Not sure If I like that we mention Magento in our docs. @fashxp WDYT?

Reply:
Changed to "web shop"